django_bmemcached/memcached.py

```diff
@@ -1,14 +1,43 @@
 import os
+
+import bmemcached
 from django.core.cache.backends import memcached
 
+# keys that are acceptable in the Django
+# settings.CACHES[...]["OPTIONS"] dictionary.
+VALID_CACHE_OPTIONS = (
+    "username",
+    "password",
+    "compression",
+    "socket_timeout",
+    "pickler",
+    "unpickler",
+    "pickle_protocol",
+)
+
+
+class InvalidCacheOptions(Exception):
+    """Custom error raised when Client initialisation fails."""
+
+    def __init__(self, options):
+        keys = options.keys()
+        invalid_options = ", ".join(
```

Comment:
this can be simplified with a set, change VALID_CACHE_OPTIONS from () to {} and then invalid_options become:

```python
invalid_options = keys - VALID_CACHE_OPTIONS
````

Reply:
Ha - so that solution is much cleaner (thx) - however it does fail CI on Python 2.7 - `unsupported operand type(s) for -: 'list' and 'set'`. I can fix this by casting the list to a set - or the nuclear option is to drop Python 2.7 support? (Big call, I know).

Reply:
As not even django itself supports python 2, I would drop it in the future but leave with the cast for now, it should be ok ;)